synchronizer/local.go

```diff
@@ -61,21 +64,25 @@ func (s *Local) SyncAll() error {
 
 // StartPeriodicFetching starts periodic fetchers tasks
 func (s *Local) StartPeriodicFetching() {
-	if s.splitTasks.SplitSyncTask != nil {
-		s.splitTasks.SplitSyncTask.Start()
-	}
-	if s.splitTasks.SegmentSyncTask != nil {
-		s.splitTasks.SegmentSyncTask.Start()
+	if s.refreshEnabled {
```

Comment:
I don't think that you need to check for this if the tasks are nil

Reply:
I moved the logic to the NewLocal.